src/appMain/smartDeviceLink.ini

```diff
@@ -411,3 +411,9 @@ RpcPassThroughTimeout = 10000
 [RCModuleConsent]
 ; The period (in days) after which consent for module_id should be removed.
 PeriodForConsentExpiration = 30
+
+[Logger]
+; Write all logs in queue to file system before shutdown 
```

Comment:
the comment "write all logs in queue" may not be correct, and it is misleading

Reply:
Fixed